src/index.js

```diff
@@ -1,8 +1,18 @@
 import $ from 'jquery';
 import runGame from './game/game-animation';
 import simulation from './lib/simulation';
+import { Simulator } from './lib/Simulator.ts';
+import { US } from './lib/Scenarios.ts';
 
 $(window).on('load', () => {
+    const simulator = new Simulator(US);
```

Comment:
Example usage of the simulator just to see if it works without errors. This will be removed in the future.